frontends/main/src/app-pages/HomePage/HomePage.tsx

```diff
@@ -41,26 +41,30 @@ const MediaCarousel = styled(ResourceCarousel)(({ theme }) => ({
 const HomePage: React.FC = () => {
   return (
     <>
-      <LearningResourceDrawer />
+      <Container>
```

Comment:
Moving this container broke some stuff. Mistake?

<img width="952" alt="Screenshot 2024-10-31 at 1 21 26 PM" src="https://github.com/user-attachments/assets/7eae4551-4e83-4e66-9f1f-0ccfc4a447d8">

Reply:
Oof, yes.